pandas/core/frame.py

```diff
@@ -9620,7 +9623,7 @@ def merge(
         left_index: bool = False,
         right_index: bool = False,
         sort: bool = False,
-        suffixes: Suffixes = ("_x", "_y"),
+        suffixes: Suffixes | None = ("_x", "_y"),
```

Comment:
`Suffixes` includes `None`, so could remove this or keep the alias in pandas._typing unchanged.